flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -522,82 +462,44 @@ int32_t vtol_follower_control_attitude(float dT)
 	float forward_accel_desired = -northCommand * cosf(yaw * DEG2RAD) + -eastCommand * sinf(yaw * DEG2RAD);
 	float right_accel_desired = -northCommand * sinf(yaw * DEG2RAD) + eastCommand * cosf(yaw * DEG2RAD);
 
+	StabilizationDesiredData stabDesired;
+
 	// Set the angle that would achieve the desired acceleration given the thrust is enough for a hover
 	stabDesired.Pitch = bound_min_max(RAD2DEG * atanf(forward_accel_desired / GRAVITY),
-	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch);
+	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch) + att_adj[1];
 	stabDesired.Roll = bound_min_max(RAD2DEG * atanf(right_accel_desired / GRAVITY),
-	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch);
+	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch) + att_adj[0];
```

Comment:
Yah, I thought about that some.  Let me think about it a little more.

I don't really suggest an att_adj that high, and I think the position controller will be fighting rather than augmenting a big roll/pitch like that without insane gains (at least in the long term; it might help in the very short acceleration phase).

Also these should probably be symmetric bounds.  I'm not sure why this is this way.